parser.js

```diff
@@ -393,6 +393,7 @@
         max: max,
         greedy: true,
         body: null, // set later on
+        symbol: symbol,
```

Comment:
Not sure, is `symbol` a good name here? How about "quantifier"?

Reply:
Yeh naming is hard. I tried searching around to find an official name for this thing but couldn't find anything :/

Quantifier is already the name that's used for this node though and the TypeScript type for this node so might be confusing?